owncloudData/src/main/java/com/owncloud/android/data/files/repository/OCFileRepository.kt

```diff
@@ -79,60 +80,70 @@ class OCFileRepository(
         }
     }
 
-    override fun copyFile(listOfFilesToCopy: List<OCFile>, targetFolder: OCFile) {
+    override fun copyFile(listOfFilesToCopy: List<OCFile>, targetFolder: OCFile, replace: List<Boolean?>): List<OCFile> {
         val sourceSpaceWebDavUrl = localSpacesDataSource.getWebDavUrlForSpace(listOfFilesToCopy[0].spaceId, listOfFilesToCopy[0].owner)
         val targetSpaceWebDavUrl = localSpacesDataSource.getWebDavUrlForSpace(targetFolder.spaceId, targetFolder.owner)
+        val filesNeedsAction = mutableListOf<OCFile>()
 
-        listOfFilesToCopy.forEach { ocFile ->
+        listOfFilesToCopy.forEachIndexed forEach@{ position, ocFile ->
```

Comment:
Because there is a return@forEach in line 131 and it's going to stops working if I don't added it [https://kotlinlang.org/docs/returns.html#break-and-continue-labels](url)